libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -41,6 +40,7 @@ bool AC_AutoTune::init_internals(bool _use_poshold,
     ahrs_view = _ahrs_view;
     inertial_nav = _inertial_nav;
     motors = AP_Motors::get_singleton();
+    const uint32_t now = AP_HAL::millis();
```

Comment:
Why moving it here?
Line 73 seams a better, more local candidate for this.

Reply:
I did that initially and I got that error when you define a variable in a case statement without the {}. So I figured I would more it to the top where we normally have it and not put {} on that case statement.

Reply:
OH, got it, thanks.